File: C/Basics/Function/program2.c
#include <stdio.h>

double add(double num1, double num2); // include function prototype

int main()
{
    double c = add(2.5, 2.5);
    printf("%lf\n", c); // 5.000000

    return 0;
}   

// int add(int num1, int num2) 
double add(double num1, double num2)
{
    double sum = num1 + num2;
    return sum;
}

File: JavaScript/Intermediate/Classes/program2.js
// define a class using class expression
// Redeclare class(A class can be declared once only. If we try to declare class more than one time it throws an error)
let Add = class{
    num1 = 100
    num2 = 20
    addTwo(){
        return this.num1 + this.num2;
    }
}

// let Add = class{
//     num1 = 100
//     num2 = 200
//     addTwo(){
//         return this.num1 + this.num2;
//     }
// }

let personObj = new Add();
console.log(personObj); // SyntaxError: Identifier 'Add' has already been declared
